core/src/main/java/com/yahoo/imapnio/async/response/ImapResponseMapper.java

```diff
@@ -378,15 +378,14 @@ private Status parseToStatus(@Nonnull final IMAPResponse[] r) throws ParsingExce
         }
 
         /**
-         * Parses the ID responses to a @{code IdResult} object.
+         * Parses the ID responses to a {@link IdResult} object.
          *
-         * @param r the list of responses from ID command, the input responses array should contain the tagged/final one
+         * @param ir the list of responses from ID command, the input responses array should contain the tagged/final one
```

Comment:
This param mismatch should be caught by check style violation, one will be "Unused @param tag for r" and the other would be "Method Javadoc: Expected @param tag for 'ir'.", unless we miss declaring some check style in checkstyle.xml .  Could you please check and fix checkstyle.xml

Reply:
Currently, our check style only checks public method javadoc.